### ISSUE TITLE
Create component => src/components/TeamMemberCard.tsx

### ISSUE THREAD

**Issue by kleneway** [PioneerSquareLabs/otto-playground#156]
Create a React component for src/components/TeamMemberCard.tsx. Here are the instructions:

### Background
We need to create a new React component called `TeamMemberCard.tsx` that will be responsible for displaying team member information in our project management application. This component will be used in the project page to show the list of team members associated with a particular project.

### Task
Create a new React component called `TeamMemberCard.tsx` in the `src/components` directory. This component will be written in TypeScript using ES6 syntax, like arrow functions.

### Component Props
The `TeamMemberCard` component should accept the following props:

```typescript
interface TeamMemberCardProps {
  teamMember: TeamMember;
}
```

The `TeamMember` type is already defined in the `types.ts` file and should be imported from there.

### Component Structure
The `TeamMemberCard` component should have the following structure:

- A container element (e.g., a `div`) with a CSS class `team-member-card` for styling purposes.
- Inside the container element, display the team member's name and email address.

### Imports
Make sure to import the necessary dependencies at the top of the `TeamMemberCard.tsx` file:

```typescript
import React from 'react';
import { TeamMember } from '~/types';
import '~/styles/team-member-card.css';
```

### State Management
There is no need for state management in this component, as it will only display the team member information passed through the props.

### Rendering
The component should render the team member's name and email address as follows:

```jsx
<div className="team-member-card">
  <h3>{teamMember.name}</h3>
  <p>{teamMember.email}</p>
</div>
```

### User Interaction
There is no user interaction required for this component.

### Styling
Create a new CSS file called `team-member-card.css` in the `src/styles` directory. Add any necessary styles for the `team-member-card` class to make the component visually appealing and consistent with the rest of the application.

### Testing
Make sure to test the `TeamMemberCard` component by using it in the project page and passing a sample `TeamMember` object as a prop. Verify that the component renders the team member's name and email address correctly.

### Error Cases
There are no error cases to handle in this component, as it only displays the team member information passed through the props. However, make sure that the component can handle cases where the `teamMember` prop is `null` or `undefined`.

### Summary
In summary, you need to create a new `TeamMemberCard.tsx` component that accepts a `TeamMember` object as a prop and displays the team member's name and email address. Make sure to import the necessary dependencies, define the component structure, and add any required styling. Test the component in the project page to ensure it renders correctly.
